packages/lit-dev-content/samples/tutorials/custom-attribute-converter/tutorial.json

```
@@ -8,26 +8,26 @@
   "imgAlt": "<my-calendar selected-date=\"03/24/22\"></my-calendar>",
   "steps": [
     {
-      "title": "Custom Attribute Converters",
+      "title": "Custom attribute converters",
       "hasAfter": true
     },
     {
-      "title": "Built-in String Attribute Converter",
+      "title": "Built-in string attribute converter"
```

This is breaking build

```suggestion
      "title": "Built-in string attribute converter",
```